docs/en/latest/plugins/proxy-mirror.md

```diff
@@ -74,85 +76,70 @@ admin_key=$(yq '.deployment.admin.admin_key[0].key' conf/config.yaml | sed 's/"/
-}'
-```
-
-### Specify the timeout for mirror subrequests
+### Mirror Partial Traffic
 
-We can specify the `timeout` for subrequests in `plugin_attr` in `conf/config.yaml`. This is useful in connection reuse scenarios when mirroring traffic to a very slow backend service.
+The following example demonstrates how you can configure `proxy-mirror` to mirror 50% of the traffic to a Route and forward them to another Upstream service.
 
-Since mirror requests are implemented as sub-requests, delays in sub-requests will block the original request until the sub-requests are completed. So you can configure the timeout time to protect the sub-requests from excessive delays that affect the original requests.
+Start a sample NGINX server for receiving mirrored traffic:
 
-| Name | Type | Default | Description |
-| --- | --- | --- | --- |
-| connect | string | 60s | Connection timeout for mirror request to upstream. |
-| read | string | 60s | The time that APISIX maintains the connection with the mirror server; if APISIX does not receive a response from the mirror server within this time, the connection is closed. |
-| send | string | 60s | The time that APISIX maintains the connection with the mirror server; if APISIX does not send a request within this time, the connection is closed. |
-
-```yaml
-plugin_attr:
-  proxy-mirror:
-    timeout:
-      connect: 2000ms
-      read: 2000ms
-      send: 2000ms
+```shell
+docker run -p 8081:80 --name nginx nginx
 ```
 
-## Example usage
-
-:::tip
+You should see NGINX access log and error log on the terminal session.
 
-For testing you can create a test server by running:
+Open a new terminal session and create a Route with `proxy-mirror` to mirror 50% of the traffic:
 
 ```shell
-python -m http.server 9797
+curl "http://127.0.0.1:9180/apisix/admin/routes" -X PUT \
+  -H "X-API-KEY: ${admin_key}" \
+  -d '{
+    "id": "traffic-mirror-route",
+    "uri": "/get",
+    "plugins": {
+      "proxy-mirror": {
+        "host": "http://127.0.0.1:8081",
+        "sample_ratio": 0.5
+      }
+    },
+    "upstream": {
+      "nodes": {
+        "httpbin.org": 1
+      },
+      "type": "roundrobin"
+    }
+  }'
 ```
 
-:::
-
-Once you have configured the Plugin as shown above, the requests made will be mirrored to the configured host.
+Send Generate a few requests to the route:
 
 ```shell
-curl http://127.0.0.1:9080/hello -i
+curl -i "http://127.0.0.1:9080/get"
 ```
 
-```shell
-HTTP/1.1 200 OK
-...
-hello world
+You should receive `HTTP/1.1 200 OK` responses for all requests.
+
+Navigating back to the NGINX terminal session, you should see a number of access log entries, roughly half the number of requests generated:
+
+```text
+172.17.0.1 - - [29/Jan/2024:23:11:01 +0000] "GET /get HTTP/1.1" 404 153 "-" "curl/7.64.1" "-"
 ```
 
-## Delete Plugin
+This suggests APISIX has mirrored the request to the NGINX server. Here, the HTTP response status is `404` since the sample NGINX server does not implement the route.
```

Comment:
```suggestion
This suggests APISIX has mirrored the request to the NGINX server. Here, the HTTP response status is `404` since the sample NGINX server does not implement the Route.
```